Plots/Masking/NCL_mask_2.py

```diff
@@ -0,0 +1,83 @@
+"""
+NCL_mask_2.py
+===============
+This script illustrates the following concepts:
+   - Using keyword zorder to mask areas in a plot
+   - Drawing filled land areas on top of a contour plot
+
```

Comment:
I would mention why you used a different color scheme in this projection. I added a comment in the concepts illustrated section of [lcnative 1](https://github.com/NCAR/GeoCAT-examples/blob/master/Plots/MapProjections/NCL_lcnative_1_lg.py) that you may choose to copy over, or even just a short line about using best practices when choosing color schemes would be fine